dbcon/joblist/tuple-bps.cpp

```diff
@@ -3382,7 +3402,8 @@ void TupleBPS::abort_nolock()
         }
 
         BPPIsAllocated = false;
-        fDec->shutdownQueue(uniqueID);
+//        fDec->shutdownQueue(uniqueID);
```

Comment:
Comment here?

Reply:
You will notice that the next line calls
    fdec->removeQueue(), which includes shutdownQueue functionality -- it's redundant. I removed the comment code.